src/adapters/MeetingsJSONAdapter.js

```diff
@@ -22,28 +26,29 @@ import {MeetingsAdapter} from '@webex/component-adapter-interfaces';
  */
 
 /**
- * Implements the MeetingsAdapter interface with a JSON object as its datasource. See @MeetingsJSON
+ * Implements the MeetingsAdapter interface with a JSON object as its datasource.
+ *
+ * @see {@link MeetingsJSON} for example datasource.
+ * @class
+ * @implements {MeetingsAdapter}
  */
 export default class MeetingsJSONAdapter extends MeetingsAdapter {
   /**
-   * Returns a MediaStream object obtained from local user media.
-   * @returns {MediaStream}
+   * Creates a new MeetingJSONAdapter.
+   *
+   * @param {object} datasource An object that contains a set of meetings keyed by ID.
+   * @hideconstructor
    */
-  async getLocalVideo() {
-    const constraints = {
-      video: true,
-      audio: false,
-    };
-    let stream;
+  constructor(datasource) {
+    super(datasource);
 
-    try {
-      stream = await navigator.mediaDevices.getUserMedia(constraints);
-    } catch (reason) {
-      // eslint-disable-next-line no-console
-      console.error('Meetings JSON adapter can not display the user local stream', reason);
-    }
+    this.meetingControls = {};
 
-    return stream;
+    this.meetingControls[MUTE_AUDIO_CONTROL] = {
```

Comment:
can you just combine the line 45 with 47 to define the `meetingControls` in one place?

Reply:
In order to use the constant for the attribute name, I need to use the `[]` notation

Reply:
I like the separation of having each control defined in its own code block.